analogway_livecore.js

```diff
@@ -1,743 +1,574 @@
+							this.modelname = 'Ascender 32 - 4K'
+							break
+						case 108:
+							this.modelname = 'Ascender 48 - 4K'
+							break
+						case 112:
+							this.modelname = 'Ascender 16'
+							break
+						case 113:
+							this.modelname = 'Ascender 16 - 4K'
+							break
+						case 114:
+							this.modelname = 'Ascender 48 - 4K - PL'
+							break
+						case 115:
+							this.modelname = 'Output Expander 48 - 4K  - PL'
+							break
+						case 116:
+							this.modelname = 'NeXtage 08'
+							break
+						case 117:
+							this.modelname = 'NeXtage 08 - 4K'
+							break
+						case 118:
+							this.modelname = 'Ascender 32 - 4K -PL'
+							break
+						case 119:
+							this.modelname = 'Output Expander 32 - 4K - PL'
+							break
+						default:
+							this.modelname = 'unknown'
+							break
+					}
+					this.log('info', this.config.label + ' Type is ' + this.modelname)
+					this.sendcmd('0,TPver')
+				}
 
-			if (line.match(/TPdie0/)) {
-				//There is no parameter readback runnning, it can be started now
-			}
+				if (line.match(/TPver\d+/)) {
+					var commandSetVersion = parseInt(line.match(/TPver\d+,(\d+)/)[1])
+					this.log('info', 'Command set version of ' + this.config.label + ' is ' + commandSetVersion)
+					// TODO: Should check the machine state now, will be implemented after feedback system is done
+				}
 
+				if (line.match(/TPdie0/)) {
+					//There is no parameter readback runnning, it can be started now
+				}
 
-			if (line.match(/E\d{2}/)) {
-				switch (parseInt(line.match(/E(\d{2})/)[1])) {
-					case 10: self.log('error',"Received command name error from "+ self.config.label +": "+ line); break;
-					case 11: self.log('error',"Received index value out of range error from "+ self.config.label +": "+ line); break;
-					case 12: self.log('error',"Received index count (too few or too much) error from "+ self.config.label +": "+ line); break;
-					case 13: self.log('error',"Received value out of range error from "+ self.config.label +": "+ line); break;
-					default: self.log('error',"Received unspecified error from Livecore "+ self.config.label +": "+ line);
+				if (line.match(/E\d{2}/)) {
+					switch (parseInt(line.match(/E(\d{2})/)[1])) {
+						case 10:
+							this.log('error', 'Received command name error from ' + this.config.label + ': ' + line)
+							break
+						case 11:
+							this.log('error', 'Received index value out of range error from ' + this.config.label + ': ' + line)
+							break
+						case 12:
+							this.log(
+								'error',
+								'Received index count (too few or too much) error from ' + this.config.label + ': ' + line
+							)
+							break
+						case 13:
+							this.log('error', 'Received value out of range error from ' + this.config.label + ': ' + line)
+							break
+						default:
+							this.log('error', 'Received unspecified error from Livecore ' + this.config.label + ': ' + line)
+					}
 				}
-			}
 
-		});
+				if (line.match(/TAopr\d+,(0|1)$/)) {
```

Comment:
Still not a Regex which will exclusive be triggered by the TAopr command. Try renaming a memory to TAopr1,1 and see what happens.